Codecademy/gamut
# feat(Flyout): focus container on open

### Overview


Moves focus to the container of the Flyout on open +  exports focus-visible utility style


### PR Checklist

- [ ] Related to designs:  no
- [x] Related to JIRA ticket: [gm-849]

#### Testing Instructions



1. Head to the Flyout story
2. Using your keyboard, open the Flyout 
3. Focus should be on the Flyout container
4. Press `tab`
5. Focus should move to the close button

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/38216)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/7226)  | [Portal Env](http://www.google.fr/ 'Named link title')   |
